cli/lib/kontena/cli/plugins/uninstall_command.rb

```diff
@@ -6,9 +6,11 @@ class UninstallCommand < Kontena::Command
     include Kontena::Cli::Common
 
     parameter 'NAME', 'Plugin name'
+
     option "--force", :flag, "Force remove", default: false, attribute_name: :forced
 
     def execute
+      exit_with_error "Plugin #{pastel.cyan(name)} is not installed" unless Kontena::PluginManager.instance.installed(name) && !ENV['NO_PLUGINS']
```

Comment:
It's there to avoid the confirm dialog when the plugin isn't installed in the first place.